app/api/v1/router.py

```diff
@@ -20,3 +21,4 @@
     prefix="/users/{user_id}/comprehensive_evaluations",
     tags=["comprehensive_evaluations"],
 )
+router.include_router(todo.router, prefix="/todos", tags=["todos"])
```

Comment:
@Ryan-OH 
여기도 아래처럼 하면 어떨지 제안 드립니다!

@GideokKim 
```suggestion
router.include_router(todo.router, prefix="/users/{user_id}/categories/{category_id}/todos", tags=["todos"])
```

Reply:
수정했슴다!